doc/enterprise-shield/o_enterprise_shield_checklist.md

```diff
@@ -0,0 +1,55 @@
+-   [Home](../../index.md)
+-   [Documentation](../index.md)
+-   Configure Enterprise Shield™ for KAAZING Gateway
+
+Configure Enterprise Shield™ for KAAZING Gateway ![This feature is available in KAAZING Gateway - Enterprise Edition](../images/enterprise-feature.png)
+==========================================================================================
+
+This checklist provides the steps necessary to configure Enterprise Shield™ in your KAAZING Gateway architecture.
+
+| #   | Step                                                            | Topic or Reference                                                                   |
+| --- | --------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
+| 1   | Become familiar with Gateway topologies                         | [Common Gateway Production Topologies](../admin-reference/c_topologies.md)   |
+| 2   | Understand how Enterprise Shield™ works                         | [About Enterprise Shield™](#about-enterprise-shield)                                 |
+| 3   | Determine which Enterprise Shield™ topology is for you          | [Common Enterprise Shield™ Use Cases](c_enterprise_shield_use_cases.md)              |
+| 4   | Download and set up KAAZING Gateway                             | [Setting Up the Gateway](../about/setup-guide.md)                                    |
+| 5   | Configure Enterprise Shield™                                    | [Walkthrough: Configure Enterprise Shield™](p_enterprise_shield_config.md)                          |
+| 6   | Configure Enterprise Shield™ for high availability and failover | [Walkthrough: Configure Enterprise Shield™ for High Availability](p_enterprise_shield_cluster.md)   |
+
+
+About Enterprise Shield™
+-----------------------------------------------
+
+Enterprise Shield™ protects your trusted network by initiating the connection from the internal trusted network towards the DMZ. Until now, allowing access to a trusted network while still maintaining security behind the firewall has been impossible because of the necessity to open ports to accept incoming connections. For most administrators, opening a port to the outside world is a necessary but undesirable solution because it instantly increases vulnerability to outside hacks and attacks. Companies are reluctant to open up ports in firewalls because each open port is another potential attack vector for malicious users. Using Enterprise Shield™, you can close all of your inbound ports while still allowing clients to initiate connections.
```

Comment:
> ... has been impossible ...

We shouldn't say impossible, it sounds too strong. How about something like "still has challenges"?